template/src/extension.ts

```diff
@@ -32,6 +32,11 @@ const createImplementationPanel = (
     selector: sourcegraph.DocumentSelector,
     providers: SourcegraphProviders
 ): void => {
+    // `createPanelView` was removed in https://github.com/sourcegraph/sourcegraph/pull/50451
+    if (!sourcegraph.app.createPanelView) {
+        return
+    }
```

Comment:
I'm not 100% sure about this approach. 
How is the implementations panel used (can we remove it)?